friends/src/Reducers/index.js

```diff
@@ -0,0 +1,37 @@
+import {LOGIN_START,LOGIN_SUCCESS,LOADING_FRIENDS} from "../Actions"
+
+const initalState={
+    loggedIn:false,
+    loadingFriends:false,
+    friends:[],
+    error:null,
+}
+
+export const reducer =(state=initalState,action)=>{
+    switch(action.type){
+    case LOGIN_START:
+    return {
+        ...state,
+        loggedIn:true,
+        error:'',
+    }
+    case LOGIN_SUCCESS:
+    return{
+        ...state,
+        loadingFriends:false,
+        friends:action.payload,
+        error:'',
```

Comment:
Once you fix the mixup between action types for the action creators file (also indicated in other code comments), you will not need to set `friends` here anymore.